src/fox/Fox.js

```diff
@@ -0,0 +1,16 @@
+import {Animal} from '../animal.js';
+import foxImage from './fox.jpg'
+
+export class Fox extends Animal {
+    constructor() {
+        super();
+        this.initiative = 7;
+        this.strenght = 4;
```

Comment:
```suggestion
        this.strength = 4;
```
Please change it everywhere